packages/babel-template/src/literal.ts

```diff
@@ -45,41 +45,33 @@ function buildLiteralData<T>(
   tpl: Array<string>,
   opts: TemplateOpts,
 ) {
-  let names;
-  let nameSet: Set<string>;
-  let metadata;
-  let prefix = "";
+  let prefix = "BABEL_TPL$";
```

Comment:
If we are always adding enough `$`s anyway, why do we need a more complex prefix?

Reply:
Just to reduce the possibility of duplication.